rialto_airflow/harvest/openalex.py

```diff
@@ -195,4 +196,17 @@ def _clean_dois_for_query(dois: list[str]) -> list[str]:
 
 
 def _doi_log_message(doi: str):
-    logging.debug(f"dropping {doi} from openalex lookup")
+    logging.warning(f"dropping {doi} from openalex lookup")
+
+
+@cache
```

Comment:
Queries OpenAlex sources for an ISSN when to get publisher data or a journal name when not already available in openalex_json.